modules/scripts/startup-script/files/running-script-warning.sh

```diff
@@ -13,17 +13,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-WARNING=$(
-	cat <<EOF
-** NOTICE **: System services may not be running until startup scripts complete.
-The output of the command below will end with "Finished Google Compute Engine 
-Startup Scripts." when they are complete. Please review the output for any 
-errors which may indicate that services are unhealthy. 
+SCRIPT_COMPLETE_FILE="/run/startup_script_msg"
 
-sudo journalctl -b 0 -u google-startup-scripts.service
-EOF
+WARNING_MESSAGE=$(
+	cat <<-EOF
+		** NOTICE **: HPC-Toolkit startup scripts should start soon.
```

Comment:
- No dash in "HPC Toolkit".
- Do not say "should" or "soon" which conveys uncertainty. Say "HPC Toolkit startup scripts are executing". Fewer, direct (and true) words.